docs/examples/index.js

```diff
@@ -95,12 +95,16 @@ function generateConfigurator(id) {
             <div class="col-md-4">
               <label for="colorSetInput" class="form-label">Categorical Color</label>
               <select class="form-select" aria-label="Color set" id="colorSetInput" onchange="changeTheme('${id}')">
-                <option value="${ODSCharts.ODSChartsCategoricalColorsSet.DEFAULT_SUPPORTING_COLORS}">Default supporting colors</option>
+                <option value="${ODSCharts.ODSChartsCategoricalColorsSet.DEFAULT}">Default colors</option>
+                <option value="${ODSCharts.ODSChartsCategoricalColorsSet.EMPHASIS}">Emphasis</option>
+                <option value="${ODSCharts.ODSChartsCategoricalColorsSet.SUPPORTING_COLORS}">Supporting colors</option>
                 <option value="${ODSCharts.ODSChartsCategoricalColorsSet.LIGHTER_TINTS}">Lighter tints</option>
                 <option value="${ODSCharts.ODSChartsCategoricalColorsSet.DARKER_TINTS}">Darker tints</option>
                 <option value="${ODSCharts.ODSChartsCategoricalColorsSet.SEQUENTIAL_BLUE}">Blue</option>
                 <option value="${ODSCharts.ODSChartsCategoricalColorsSet.SEQUENTIAL_GREEN}">Green</option>
+                <option value="${ODSCharts.ODSChartsCategoricalColorsSet.SEQUENTIAL_PINK}">Pink</option>
                 <option value="${ODSCharts.ODSChartsCategoricalColorsSet.SEQUENTIAL_PURPLE}">Purple</option>
+                <option value="${ODSCharts.ODSChartsCategoricalColorsSet.SEQUENTIAL_YELLOW}">Yellow</option>
```

Comment:
This Merge Request reveal we have a bug on the legend of the number of color is less than the number of curves

By instance ODSCharts.ODSChartsSequentialColorsSet.SEQUENTIAL_BLUE has only 5 colors
![image](https://github.com/Orange-OpenSource/ods-charts/assets/40755890/f7752b08-c295-4ab1-b52e-72074d8c1da9)

Reply:
Do you prefer I fix it in a separate bug?